CHANGES.md

```diff
@@ -1,6 +1,85 @@
 Release Changes List
 ====================
 
+version 1.22.0
+==============
+
+Twenty-fifth public release of Chapel, April 16, 2020  
+Second release candidate for Chapel 2.0 (RC2)
```

Comment:
In keeping with the release's theme, shouldn't this be considered the twenty-fourth public release, and the first release candidate?

From the "the more things change" department, what should now be considered the first public release has this CHANGES entry:

```
  - open interval syntax supported for ranges
    e.g., "[lo..hi)" is equivalent to lo..hi-1
```

Reply:
I don't understand what you're saying.

Reply:
Sorry, I was just suggesting the release info should be counting from 0 now.

Reply:
Ohhhhhh... :D

>   - open interval syntax supported for ranges
>     e.g., "[lo..hi)" is equivalent to lo..hi-1

Huh...  I had completely remembered that we had discussed that idea, but had no recollection that we had actually implemented it.  Weird!  (it was rejected not because it was a bad concept so much as terrible syntax for the concept.  Then the `#` operator made its appearance and we thought our open-interval days were behind us.